qcelemental/tests/test_units.py

```diff
@@ -54,6 +54,25 @@ def test_unit_conversion_nist(from_unit, to_unit, expected):
         ("Angstrom", "Angstrom", "1.0"),
         ("coulomb", "statC", "2997924580"),
         ("debye", "e * Bohr", "0.393430273"),
+        # Derived atomic units (https://en.wikipedia.org/wiki/Hartree_atomic_units)
```

Comment:
+1 for the tests here, especially for the complex units.

Reply:
Can we add an `assert str(unit)` to the tests here based off the surprising result below?